c/zss.c

```diff
@@ -1041,6 +1041,25 @@ static void readAgentAddressAndPort(JsonObject *serverConfig, JsonObject *envCon
   }
 }
 
+static char* generateCookieName(JsonObject *envConfig, int port) {
+  int cookieLength=256;
+  char *cookieName = safeMalloc(cookieLength+1, "CookieName");
```

Comment:
It's a good practice to check for `NULL`. Perhaps we should have a wrapper for `malloc`/`safeMalloc` which would abort if NULL is returned. We could use it for places where we don't want to continue execution if such a condition occurs.